tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -106,20 +107,31 @@ suite('Language server validation', async () => {
         await checkJavaPathCompletion();
     });
 
-    test('Error highlighting', async () => {
-        await editor.type(javaFileName, 'error', 30);
-        await editor.waitErrorInLine(30, TimeoutConstants.TS_ERROR_HIGHLIGHTING_TIMEOUT * 3);
-        await editor.performKeyCombination(javaFileName, Key.chord(Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE, Key.BACK_SPACE));
-        await editor.waitErrorInLineDisappearance(30);
-    });
-
     test('Autocomplete', async () => {
         await editor.moveCursorToLineAndChar(javaFileName, 32, 17);
         await editor.pressControlSpaceCombination(javaFileName);
         await editor.waitSuggestionContainer();
         await editor.waitSuggestion(javaFileName, 'SpringApplication - org.springframework.boot');
     });
 
+    test('Error highlighting', async () => {
+        await editor.type(javaFileName, '$#%@#', 30);
+        try {
+            await editor.waitErrorInLine(30, TimeoutConstants.TS_ERROR_HIGHLIGHTING_TIMEOUT * 3);
+        } catch (err) {
+            Logger.debug('==============>>>>>>>>>>>>>>>>>> Workaround for error highlighting.');
```

Comment:
It would be nice to have the link to related issue https://github.com/eclipse/che/issues/18974 in the code.